tox.ini

```diff
@@ -4,5 +4,4 @@ envlist = py27,py34
 deps=
     -rrequirements.txt
     pytest-cov
-    py27: mock
 commands=py.test --cov --cov-config=.coveragerc tests/robottelo
```

Comment:
MOck is already on requirements.txt, Travis updated setuptools and now the new version complains if we try to install already listed package.